<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Рецепты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="4">
         <widget class="QPushButton" name="add_recipe">
          <property name="text">
           <string>Добавить рецепт</string>
          </property>
         </widget>
        </item>
        <item row="5" column="7">
         <widget class="QPushButton" name="recipe_list_refresh">
          <property name="text">
           <string>Обновить список</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" rowspan="2" colspan="2">
         <widget class="QListWidget" name="list_ingredients_available_4"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="sort_name_down">
          <property name="text">
           <string>Я-а</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="sort_ingredients">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>По ингредиентам</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="5" colspan="3">
         <widget class="QListWidget" name="list_recipe_available"/>
        </item>
        <item row="2" column="4" colspan="4">
         <widget class="QPlainTextEdit" name="recipe_recipe"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="sort_name_up">
          <property name="text">
           <string>А-я</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="2" colspan="2">
         <widget class="QLabel" name="picture_recipe">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QPushButton" name="change_recipe">
          <property name="text">
           <string>Редактирование</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="remove_recipe">
          <property name="text">
           <string>Удалить рецепт</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Холодильник</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="1" colspan="3">
         <widget class="QPushButton" name="add_new_ingredient">
          <property name="text">
           <string>Добавить ингредиент</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="3">
         <widget class="QPushButton" name="delete_ingredient">
          <property name="text">
           <string>Удалить ингредиент</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QPushButton" name="save_ingredient_change">
          <property name="text">
           <string>Сохранить изменения</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="3">
         <widget class="QListWidget" name="ingredients_ingredients"/>
        </item>
        <item row="6" column="1" colspan="3">
         <widget class="QPushButton" name="refresh_list_ingredients">
          <property name="text">
           <string>Обновить список</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Колличество выбранного ингредиента</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QSpinBox" name="ingredient_count">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="ingredient_name">
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Имя ингредиента</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
